### PR TITLE
fix: alchemy package.json was incorrectly pointing to files

### PR DESCRIPTION
**Pull request by moldy530**
# Pull Request Checklist

- [ ] Did you add new tests and confirm existing tests pass? (`yarn test`)
- [ ] Did you update relevant docs? (docs are found in the `site` folder, and guidelines for updating/adding docs can be found in the [contribution guide](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md))
- [ ] Do your commits follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] Does your PR title also follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] If you have a breaking change, is it [correctly reflected in your commit message](https://www.conventionalcommits.org/en/v1.0.0/#examples)? (e.g. `feat!: breaking change`)
- [ ] Did you run lint (`yarn lint:check`) and fix any issues? (`yarn lint:write`)
- [ ] Did you follow the [contribution guidelines](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: packages/core/package.json [hunk 1]
```diff
@@ -27,13 +27,6 @@
     },
     "./package.json": "./package.json"
   },
-  "typesVersions": {
```
Comment (moldy530) [inline packages/core/package.json:30]: same here

File: packages/alchemy/package.json [hunk 2]
```diff
@@ -23,23 +23,11 @@
...
       "types": "./dist/types/index.d.ts",
-      "import": "./dist/esm/index.js",
-      "default": "./dist/cjs/index.js"
-    },
-    "./viem": {
-      "types": "./dist/types/viem/index.d.ts",
-      "import": "./dist/esm/viem/index.js",
-      "default": "./dist/cjs/viem/index.js"
+      "import": "./dist/esm/src/index.js",
+      "default": "./dist/cjs/src/index.js"
```
Comment (moldy530) [inline packages/alchemy/package.json:27]: I could not tell you what's different about aa-alchemy and aa-core that aa-alchemy is spitting out files into `src` for the esm and cjs files

**Review by denniswon — APPROVED** [review]
Can you summarize what changed on viem v2 package settings that's causing this?

**Comment by denniswon** [comment]
Seems weird it's only alchemy. Maybe some conflict with alchemy-sdk pkg

**Comment by moldy530** [comment]
ah @dphilipson figured it out. I added a reference to `../package.json` at somepoint. I need to update that to use a `version.ts` file similar to how core does it

**Review by dphilipson — APPROVED** [review]
Looks great!
